macros_doc.md

```diff
@@ -9,15 +34,40 @@ such as Excel.
 
 MkDocs (the tool we use to turn the markdown version of the specification into
 HTML pages) supports macros. In the BIDS specification document, we use these
-macros to standarize the format of items such as tables and examples.
+macros to standardize the format of items such as tables and examples.
+
+## What kind of input information are required by macros?
+
+It will depend for each macro.
```

Comment:
Replace lines 41-45 with:

Some macros only use the arguments you directly supply in the macro call.